example/advertisement.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:dbus/dbus.dart';
+import 'package:bluez/bluez.dart';
+
+void main() async {
+  var client = BlueZClient();
+  await client.connect();
+
+  var adapter = client.adapters.first;
+  var advertMngr = adapter.advertisingManager;
+
+  var advert = BlueZAdvertisement(
+    DBusObjectPath('/org/bluez/example/advertisement0099'),
```

Comment:
We don't want to provide this.

Reply:
Alright, it is automatically generated now.